scripts/rust_ci_helper.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+import json
+import argparse
+import subprocess
+import os
+
+
+def run(command):
+    assert isinstance(command, str)
+    print(f"Running: {command}")
+    exit_code = subprocess.check_call(command, shell=True, text=True)
```

Comment:
Even if you don't use the output (both stdout/err) from the command, I would suggest it's still useful to print it in case something goes wrong.